docs/scos/dev/front-end-development/202212.0/oryx/reactivity/signals.md

```diff
@@ -0,0 +1,103 @@
+# Oryx Framework - Signals
+
+## Overview
+
+Signals are a crucial tool within the Oryx framework, offering a clean and efficient reactivity API for components. They allow for the declarative formulation of component logic while seamlessly integrating with observables from domain services.
+
+## Implementation of Signals in Oryx
+
+The Oryx implementation of signals has a core mechanism and a simplified API. The core is well-suited for advanced usage, while the simplified API is sufficiently robust for most components. This documentation will focus on the simplified API.
+
+### Creating Signals
+
+To create a signal, use the `signal()` function. This function can either take a raw value or accept an observable.
+
+Here's an example of creating a simple signal:
+
+```ts
+const counter = signal(1);
+```
+
+Changing the signal value:
+
+```ts
+counter.set(2);
+```
+
+Creating a signal from an observable:
+
+```ts
+const values = signal(observable$);
+```
+
+You can initialize signals with options to adjust their behavior:
+
+- `equal` function:This option allows you to compare two values and determine their equality. It's a function that takes two inputs and returns true if they're equal and false if not. This is particularly useful when you want to control when the signal updates. By defining how values are considered equal, you can prevent unnecessary updates when the new value is effectively the same as the old one, according to your equality logic. This can help improve performance by reducing redundant computations and re-renders.
```

Comment:
```suggestion
- `equal` function: This option allows you to compare two values and determine their equality. It's a function that takes two inputs and returns true if they're equal and false if not. This is particularly useful when you want to control when the signal updates. By defining how values are considered equal, you can prevent unnecessary updates when the new value is effectively the same as the old one, according to your equality logic. This can help improve performance by reducing redundant computations and re-renders.
```

Is there no standard comparison done to detect if the new value equals the previous value?

Reply:
there is, updated